src/components/Accordion/Accordion.tsx

```diff
@@ -21,6 +21,8 @@ import {
   ShorthandValue,
   ShorthandRenderFunction,
 } from '../../../types/utils'
+import { childrenDependentRtlAttributes } from '../../lib/rtl'
+import { RtlFunc } from '../../lib/rtl/types'
```

Comment:
would expect to consume both from single module - could you, please, suggest why this separation is done?

Reply:
Will export the types from the index file as well.